README.md

```diff
@@ -1,8 +1,15 @@
+
 # dmap-import
 Import DMAP data into a PostgreSQL database
 
-## Dev Setup
-* Install ASDF ([installation guide](https://asdf-vm.com/guide/getting-started.html))
+## Dev Setup (Mac)
+
+* Install ASDF, Python3, Postgres
+```sh
+brew install asdf
+brew install python3
+brew install postgresql
```

Comment:
the python install should be handled by `asdf`

is `postgresql` here for using `psql` from the cli?

Reply:
> is postgresql here for using psql from the cli?

yes. and to build psycopg2 binary.

> the python install should be handled by asdf

for some reason it wasn't working for me. the local python3 that MacOS has installed was interfering.